src/components/shared/SubscriberHeader/SubscriberHeader.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { Loading } from '../../Loading/Loading';
+import SafeRedirect from '../../SafeRedirect';
+import SubscriberInfo from '../../shared/SubscriberInfo/SubscriberInfo';
+
+const SubscriberHeader = ({ email, dependencies: { dopplerApiClient } }) => {
+  const [stateSubscriber, setStateSubscriber] = useState({ loading: true });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setStateSubscriber({ loading: true });
+      const responseSubscriber = await dopplerApiClient.getSubscriber(email);
+      if (responseSubscriber.success) {
+        const subscriber = {
+          ...responseSubscriber.value,
+          firstName: responseSubscriber.value.fields.find((x) => x.name === 'FIRSTNAME'),
+          lastName: responseSubscriber.value.fields.find((x) => x.name === 'LASTNAME'),
+        };
+
+        setStateSubscriber({
+          subscriber: subscriber,
+          loading: false,
+        });
+      } else {
+        setStateSubscriber({ redirect: true });
+      }
+    };
+    fetchData();
+  }, [dopplerApiClient, email]);
+
+  if (stateSubscriber.redirect) {
+    return <SafeRedirect to="/Lists/MasterSubscriber/" />;
+  }
+
+  return (
+    <>
+      {stateSubscriber.loading ? (
+        <Loading />
+      ) : (
+        <SubscriberInfo subscriber={stateSubscriber.subscriber} />
```

Comment:
I would trade names, why? because `SubscriberInfo` gives the idea that you are working on information as in data. 
On the other hand `SubscriberHeader` gives the idea of something graphical, like a header of a page. This is totally debatable, of course.

Reply:
Yes first time i read i thought `SubscriberHeader` its something with styles